lib/symengine.rb

```diff
@@ -29,6 +28,10 @@ def symbols ary_or_string, *params
       end
     end
   end
+  
+  def SymEngine.Function(n)
```

Comment:
You can move this inside `class << self` block and then you can remove `SymEngine.`